proposals/0000-numa-aware-work-stealing-scheduler.md

```diff
@@ -0,0 +1,57 @@
+# NUMA-Aware Work Stealing Scheduler
+
+* Proposal: [SE-0000](0000-template.md)
+* Author: [Sebastian Woelke](https://github.com/aufgang001)
```

Comment:
oO yes, we are.
Try to book a flight ...
I simply try to avoid as special characters where ever I can.